kernel/src/main/resources/org/kframework/utils/file/README.md

```diff
@@ -0,0 +1,45 @@
+---
+copyright: Copyright (c) K Team. All Rights Reserved.
```

Comment:
Presuming #3874 is about to be merged, let's change this to `Copyright (c) Runtime Verification, Inc. All Rights Reserved.`